cmd/zpool/zpool_main.c

```diff
@@ -966,7 +968,7 @@ zpool_do_add(int argc, char **argv)
 	char *propval;
 
 	/* check options */
-	while ((c = getopt(argc, argv, "fgLno:P")) != -1) {
+	while ((c = getopt(argc, argv, "afgLno:PV")) != -1) {
```

Comment:
```suggestion
	while ((c = getopt(argc, argv, "fgLno:PV")) != -1) {
```

I wonder if either `a` or `A` still wouldn't be a more appropriate option flag here.  `V` doesn't make me think ashift.